src/mixins/itext_key_behavior.mixin.js

```diff
@@ -116,11 +120,24 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       return;
     }
     var keyMap = this.direction === 'rtl' ? this.keysMapRtl : this.keysMap;
-    if (e.keyCode in keyMap) {
-      this[keyMap[e.keyCode]](e);
+    var func;
```

Comment:
```suggestion
var keyMap = this.direction === 'rtl' ? this.keysMapRtl : this.keysMap,
    func;
```
maybe comma separate these declarations?